server/src/main/java/org/candlepin/config/ConfigProperties.java

```diff
@@ -180,21 +177,13 @@ private ConfigProperties() {
         DB_PASSWORD,
     };
 
-    // Pinsetter
-    public static final String TASKS = "pinsetter.tasks";
-    public static final String DEFAULT_TASKS = "pinsetter.default_tasks";
-    public static final String ENABLE_PINSETTER = "candlepin.pinsetter.enable";
-    public static final String PINSETTER_ASYNC_JOB_TIMEOUT = "pinsetter.waiting.timeout.seconds";
-    public static final String PINSETTER_MAX_RETRIES = "pinsetter.retries.max";
-    public static final int PINSETTER_MAX_RETRIES_DEFAULT = 10;
-
     public static final String[] DEFAULT_TASK_LIST = new String[] {};
 
     public static final String MANIFEST_CLEANER_JOB_MAX_AGE_IN_MINUTES =
```

Comment:
This is no longer used and can be removed . The property moved to `ManifestCleanerJob.CFG_MAX_AGE_IN_MINUTES` & `ManifestCleanerJob.DEFAULT_MAX_AGE_IN_MINUTES`, which is used by way of the whole job prefix/suffix system (e.g.  you'd have to write `candlepin.async.jobs.ManifestCleanerJob.max_age_in_minutes=1234` in the config to set this).

Reply:
done